app/src/main/res/layout/activity_payment_method.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+  android:id="@+id/activity_payment_methods_layout"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@color/ksr_grey_100"
+  android:orientation="vertical">
+
+  <android.support.design.widget.AppBarLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <include layout="@layout/payment_methods_toolbar" />
+
+  </android.support.design.widget.AppBarLayout>
+
+  <ScrollView
```

Comment:
Hey @dannyalright, we are wondering: should the whole screen scroll under the toolbar or only the list of cards. I'm leaning towards the list so we can always see the "Add a new card" row.